databases/week1/db_homework_week1.sql

```diff
@@ -0,0 +1,82 @@
+--Find out how many tasks are in the task table
+SELECT
+    count (*) as "task_count"
+FROM
+    task;
+--Find out how many tasks in the task table do not have a valid due date
+SELECT
+    count (*) as "task_count"
+FROM
+    task
+WHERE
+    due_date IS NULL;
+--Find all the tasks that are marked as done
+SELECT
+    task.title AS 'DONE'
+FROM
+    task
+JOIN status ON status_id = task.status_id
+WHERE
+    status.name = 'DONE';
+--Find all the tasks that are not marked as done
+SELECT
+    task.title AS 'not DONE',
+    status.name AS 'actual status'
+FROM
+    task
+JOIN status ON status_id = task.status_id
+WHERE
+    status.name != 'DONE';
+--Get all the tasks, sorted with the most recently created first
+SELECT
+    *
+FROM
+    task
+ORDER BY
+    -created;
+
+--Get the single most recently created task
+SELECT
+    *
+FROM
+    task
+ORDER BY
+    -created
+LIMIT
+    1;
+--Get the title and due date of all tasks where the title or description contains database
+SELECT
+    task.title as 'title',
+    task.due_date
+FROM
+    task
+WHERE
+    task.title LIKE '%database%' OR
+    task.description LIKE '%database%';
+--Get the title and status (as text) of all tasks
+SELECT
+    task.title,
+    status.name as 'text'
+FROM
+    task
+JOIN status ON task.status_id = status_id;
```

Comment:
Again, pay attention to the status table columns -> it's not status_id it's id, so you should use status.id